src/index.js

```diff
@@ -12,10 +12,40 @@ import { render } from '@wordpress/element';
 import wcNavFilterRootUrl from './wc-navigation-root-url';
 import LaunchStorePage from './launch-store';
 import WelcomeModal from './welcome-modal';
+import { PaymentGatewaySuggestions } from './payment-gateway-suggestions';
 import './index.scss';
 
 wcNavFilterRootUrl();
 
+if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
```

Comment:
I agree we don't need it. 

@moon0326 What do you think?

Reply:
When I wrote the rest of the checks this way, I thought because the variables were globally accessible through the `window` object, it should be a good idea to typecast again just to be more defensive. 

Please let me know your thoughts; if you think it's overkill, feel free to remove it!